_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: add_entries
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Trace peer forwarder
+
+The `trace peer forwarder` processor is used to reduce the number of events that are forwarded in a trace analytics pipeline by half when using [Peer Forwarder]({{site.url}}{{site.baseurl}}/managing-data-prepper/peer-forwarder/). It groups the events based on `trace_id` similar to `service_map_stateful` and `otel_trace_raw ` processors. 
+
+In [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/) each event is duplicated, when it is sent from `otel-trace-pipeline` to `raw-pipeline` and `service-map-pipeline`. The event is forwarded once in each pipeline. Using this processor event will be forwarded only once in `otel-trace-pipeline` to correct peer. 
+
+## Basic usage
+
+To get started with `trace peer forwarder`, create the following `pipeline.yaml` file along with [Peer forwarder]({{site.url}}{{site.baseurl}}/managing-data-prepper/peer-forwarder/) in the `data-prepper-config.yaml` file:
```

Comment:
@oeyh Can we define these parameters so the user knows how to configure this file?

Reply:
> Can we define these parameters so the user knows how to configure this file?

Do you mean `{{site.url}}{{site.baseurl}}`? I think these are embedded code. When the document gets built into webpage, the parameters will be replaced automatically.

> Where should these files be stored?

`data-prepper-config.yaml` file is one of the two configuration files described here: https://github.com/opensearch-project/data-prepper/blob/main/docs/configuration.md (the other one is pipelines.yaml that we talked about previously)